germanium/test_cases/migrations.py

```diff
@@ -39,6 +39,3 @@ def setUp(self):
 
     def set_up_before_migration(self, apps):
         pass
-
-    def test_after_migration(self):
```

Comment:
tohle tu byt nemuze, pak se spousti i ten test v predkovy kde nejsou inicializovany veci jako migration_from a pada to...